tfx/extensions/google_cloud_big_query/utils.py

```diff
@@ -30,21 +30,21 @@
 @beam.typehints.with_input_types(beam.Pipeline)
 @beam.typehints.with_output_types(beam.typehints.Dict[str, Any])
 def ReadFromBigQuery(pipeline: beam.Pipeline,  # pylint: disable=invalid-name
-                     query: str) -> beam.pvalue.PCollection:
+                     query: str,
+                     big_query_custom_configs: Optional[Dict[str, Any]]) -> beam.pvalue.PCollection:
```

Comment:
Probably, `big_query_custom_configs: Optional[Dict[str, Any]] = None` would be recommended, but I will do it by myself on the internal review!